src/bench/mempool_eviction.cpp

```diff
@@ -21,116 +21,96 @@ static void AddTx(const CTransactionRef& tx, const CAmount& nFee, CTxMemPool& po
         spendsCoinbase, sigOpCost, lp));
 }
 
-// Right now this is only testing eviction performance in an extremely small
-// mempool. Code needs to be written to generate a much wider variety of
-// unique transactions for a more meaningful performance measurement.
-static void MempoolEviction(benchmark::Bench& bench)
+void add_parents_child(std::vector<CTransactionRef>& txns, size_t package_size, size_t set_num)
+{
+    // N+1 N-input-N-output txns, in a parents-and-child package
+    // Where the final tx takes an input from all ancestors
```

Comment:
I could be reading this code wrong, but isn't each tx taking inputs from each tx before it? It's still parents-and-child technically but it's also a chain.

```suggestion
    // Where each tx takes an input from all prior txns
```